src/com/b44t/messenger/DcContact.java

```diff
@@ -50,10 +50,6 @@ public String toString() {
         return getAddr();
     }
 
-    public boolean isSeenRecently() {
-        return DateUtils.isWithin(getLastSeen(), 10, TimeUnit.MINUTES);
```

Comment:
since this was removed, now the imports introduced in:
https://github.com/deltachat/deltachat-android/pull/2371/files#diff-9ad8aa37308d8cdfde999a39743f5a70f62cc1488955eba5c677f7a299876f03R3-R6
can be removed also maybe make `DateUtils.isWithin()` private again

Reply:
great! this also makes the wrapper standalone again (so that it can be used eg. for java-bots or -extensions, iirc @angelo-fuchs was interested in that from time to time :)